src/features/customerForm/AddCustomerFormContainer.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { PureQueryOptions } from 'apollo-client';
+
+import CustomerForm from './CustomerForm';
+import { CustomerFormValues, FormProps } from './types';
+import useCreateNewCustomer from '../../common/hooks/useCreateNewCustomer';
+import { mapCustomerGroupAsOrganizationType } from './utils';
+
+export interface AddCustomerFormProps
+  extends Omit<FormProps<CustomerFormValues>, 'initialValues' | 'onCreate' | 'onDelete' | 'refetchQueries'> {
+  refetchQueries?: PureQueryOptions[] | string[];
+}
+
+const AddCustomerFormContainer = ({ onCancel, onSubmit, refetchQueries }: AddCustomerFormProps) => {
+  const createNewCustomer = useCreateNewCustomer(refetchQueries);
+
+  const handleSubmit = (values: CustomerFormValues) => {
```

Comment:
this function might be easier to read like this:

```
  const handleSubmit = (values: CustomerFormValues) => {
    createNewCustomer({
      ...values,
      companyName: values.organizationName,
      municipality: values.city,
      organizationType: mapCustomerGroupAsOrganizationType(values.customerGroup),
      phoneNumber: values.phone,
      zipCode: values.postalCode,
    });
    onSubmit?.(values);
  };
```

Reply:
Indeed it would be 👍 The `createNewCustomer` function's `customerInfo` param will contain extra values like this, but since the function picks only the ones specified in the type, this is not a problem

Reply:
Great :)